lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin`.chomp
+      if !repo.empty?
```

Comment:
But I don't want to use `unless` with `else`. 

>`if !` never reads better in Ruby :)

I disagree in this case, but I don't really mind. I'll swap the messages.